model_analyzer/config/input/config_command.py

```diff
@@ -36,7 +36,6 @@ def __init__(self):
         """
 
         self._fields = {}
-        self._fill_config()
```

Comment:
Are you sure about this change? Does our CI pass?  I know that ConfigCommand hasn't implemented it, but I think there are child classes that rely on this function call.

Reply:
I see now that you manually added the function call to all the child classes, and that the recommendation is to not call subclass functions in init.  Probably better ways to handle this with interface class or calling fill_config outside of init, but I'm fine to leave this how you have it.